src/Worker.Extensions.DurableTask/DurableTaskExtensionStartup.cs

```diff
@@ -54,6 +54,7 @@ public override void Configure(IFunctionsWorkerApplicationBuilder applicationBui
             return new DurableTaskShimFactory(options, factory); // For GrpcOrchestrationRunner
         });
 
+        applicationBuilder.Services.Configure<WorkerOptions>(o => o.InputConverters.RegisterAt<DurableTaskClientConverter>(0));
```

Comment:
Is this step necessary? I thought the registration of input converters was done via the attribute?

Reply:
Oh I need to remove that attribute haha. It doesn't work. Input binding attributes pointing to their input converter is not supported (yet).

Reply:
I see - yeah, I guess if it only works with the target class (which isn't doable in our case), then we should just remove that attribute.